internal/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"path/filepath"
+	"strconv"
+
+	"github.com/btcsuite/btcutil"
+)
+
+var (
+	DefaultName    = "nigiri.config.json"
+	DefaultDatadir = btcutil.AppDataDir("nigiri", false)
+	DefaultPath    = filepath.Join(DefaultDatadir, DefaultName)
+
+	InitialState = map[string]string{
+		"attachliquid": strconv.FormatBool(false),
```

Comment:
attachliquid could be removed if we used only one docker-compose file instead of 2.
`nigiri start` would be a wrapper of `docker-compose up -d bitcoin electrs esplora chopsticks` while `nigiri start --liquid` would be the wrapped of `docker-compose up -d`.